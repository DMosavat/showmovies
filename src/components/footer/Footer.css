
.Footer {
    margin-top: 50px;
    height: 70px;
    background-color: rgb(245, 245, 245);

}

.Footer-label {
    font-size: small;

}