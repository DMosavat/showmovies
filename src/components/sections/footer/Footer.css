
.Footer {
    margin: 10px
}

.Footer-label {
    font-size: small
}