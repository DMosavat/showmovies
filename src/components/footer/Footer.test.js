import React from 'react';
import { shallow } from 'enzyme';
import Footer from './index'

describe('Footer Test', () => {

    describe('<Footer />', () => {

        it('should render the form', () => {
            const wrapper = shallow(<Footer />);
            expect(wrapper.html()).toMatchSnapshot();
        })
    
    })
})