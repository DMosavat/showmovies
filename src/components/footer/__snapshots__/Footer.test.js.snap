// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Test <Footer /> should render the form 1`] = `"<div class=\\"Footer\\"><div class=\\"modal-body\\"><div class=\\"row justify-content-md-center\\"><label class=\\"Footer-label\\">Â© 2019 Show Movies App</label></div></div></div>"`;
